src/steps/10-FinancialDetails/IGCE/CostSummary.vue

```diff
@@ -227,6 +289,54 @@ export default class CostSummary extends Vue {
     this.createTableData(tableObject, isClinAmount,tableObject.CLINTypeClassAggregate)
   }
 
+  public async findMissingEstimates(): Promise<void>{
+    let missingCostEstimates = false
+    let hasTraining = false
+    IGCEStore.igceEstimateList.forEach(estimate=>{
+      if(estimate.unit_price === 0
+        ||estimate.unit_price === null
+        ||estimate.unit_price === undefined){
+        missingCostEstimates = true
+      }
+    })
+    const currentEnvReplicateOrOptimize = CurrentEnvironment.currentEnvironment
+      .current_environment_replicated_optimized;
+    const archDesign = DescriptionOfWork.DOWArchitectureNeeds.needs_architectural_design_services;
+    const dowObject = DescriptionOfWork.DOWObject;
+    const travel = DescriptionOfWork.travelSummaryInstances;
+    dowObject.forEach(service => {
+      if(service.serviceOfferingGroupId === "TRAINING"){
+        hasTraining = true
+      }
+    })
+    if(currentEnvReplicateOrOptimize !== "NO"
+      && currentEnvReplicateOrOptimize !== ""
+      && IGCEStore.requirementsCostEstimate?.optimize_replicate.estimated_values === []){
```

Comment:
pls change this and the other two `=== []` to `.length === 0` 
<img width="742" alt="Screen Shot 2023-05-25 at 8 27 51 PM" src="https://github.com/dod-ccpo/atat-web-ui/assets/90333349/d3c5ccfd-2d60-40d5-8d3d-d319f86634ad">